### PR TITLE
ISPN-13353 View configuration in json, xml and yaml

### PR DESCRIPTION
**Pull request by dipguptaredhat**

### CODE REVIEW

**Comment by karesti** [comment]
@dipguptaredhat I changed some code

**Review by karesti — CHANGES_REQUESTED** [review]
as discussed, create a custom hook and add unit test

**Comment by dipguptaredhat** [comment]
@karesti Need your help on this error.

Got this when running jest test on CacheConfiguration.test.tsx

Screenshot from 2022-02-02 16-30-48

File: src/__tests__/views/CacheConfiguration.test.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	</encoding>
+</distributed-cache>`;
+
+describe('Configuration Page', () => {
+  test('YAML Configuration testing', () => {
+    mockedConfigHook.useFetchConfigurationYAML.mockImplementationOnce(() => {
+      return yamlConfig;
+    });
+
+    render(
```
Comment (dipguptaredhat) [inline src/__tests__/views/CacheConfiguration.test.tsx:49]: @karesti Need your help on this error.

Got this when running jest test on CacheConfiguration.test.tsx

Screenshot from 2022-02-02 16-30-48

**Review by karesti — APPROVED** [review]

**Pull request merged**